assets/js/components/HostDetails/HostDetails.jsx

```diff
@@ -36,6 +38,18 @@ function HostDetails() {
   const { grafanaPublicUrl } = config;
 
   const [exportersStatus, setExportersStatus] = useState([]);
+  const [cleanUpModalOpen, setCleanUpModalOpen] = useState(false);
+
+  const dispatch = useDispatch();
+
+  const openDeregistrationModal = () => {
```

Comment:
I think we can use directly the `setCleanUpModalOpen(true)` , right?
I know that you did to be consistent with the other.
Anyway, up to you, not a big deal hehe